.github/workflows/appsec.yml

```diff
@@ -31,6 +31,8 @@ jobs:
       - run: yarn test:appsec:ci
       - uses: ./.github/actions/node/18
       - run: yarn test:appsec:ci
+      - uses: ./.github/actions/node/20
```

Comment:
Most of these don't need to be changed since they are tested using the oldest and newest versions we support but not the ones in-between.

Reply:
No, we should be testing all major lines as they are LTS lines which likely have a lot of users. Also, @rochdev, with the release of Node.js 21 the "latest" now _does not_ test v20 so we _do_ need to add these. Whenever an odd release happens we should be making sure the latest even release is still tested and that the new odd release is _also_ tested.

Reply:
The pattern that I am currently following to the fix is:
- if it is testing, specifically in node 16 and 18, add 20
- if it is testing only in oldest and latest, maintain it.

But with this fix, we should be aware that we are running the test in node 16 and 21. 

Other possible fix is to change the latest to 20, but I don't like the idea to not test in v21, as we have detected some things to fix there.

Reply:
I feel like we should probably have explicit version checks rather than "latest". When an odd release happens, manually add a matching version for it. When the next even version comes, replace that with that even version. 🤔